sdk/compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Error.hs

```diff
@@ -317,6 +317,12 @@ instance Pretty ErrorOrWarning where
         , "It is recommended that scripts/tests are defined in a separate package to your templates, and that"
         , "you remove `" <> pprintDep dep <> "` from the dependencies in your daml.yaml"
         ]
+    WEForbiddenNewImplementation tpl iface ->
+      vcat
+        [ "Implementation of interface " <> pPrint iface <> " by template " <> pPrint tpl <> " appears in this package, but does not appear in package that is being upgraded."
+        , "New interface instances are supported, but should only be added to supersede a different existing interface."
```

Comment:
```suggestion
```

Reply:
I found this more confusing than helpful. What's missing if it is gone?

Reply:
I added this on a bit of a whim, while trying to clarify the error message - if it's not helping you it certainly won't help users. I'll remove it.

Reply:
d36d770882